### ISSUE TITLE
Spell not found on action bars: Fire Breath(357208)

### ISSUE THREAD

**Issue by Mewohkie** [kaminaris/MaxDps-Evoker#4]
When adding or removing Fire Breath to the action bars, or using the ability and charging it up, The chat is spammed with the same message. 
"Spell not found on action bars: Fire Breath(357208)"

Fire Breath is indeed on the action bar when this error occurs.

**Comment by doadin** [comment]
I have pushed out an update to all specs, this should be fixed now.

**Comment by ArcStriker** [comment]
Mine is Showing the same thing. I have every addon updated to most recent.

**Comment by doadin** [comment]
@ArcStriker  what spec are you playing? Are you using any addon for action bars?

**Comment by doadin** [comment]
@ArcStriker

**Comment by doadin** [comment]
are you using v10.0.8 for MaxDps_Evoker and v10.0.33 for MaxDps?

**Comment by doadin** [comment]
@ArcStriker this issue should be finally fixed in v10.0.9 of this module.

**Comment by ArcStriker** [comment]
![image_2024-02-16_125919820](https://github.com/kaminaris/MaxDps-Evoker/assets/148912192/93e7f7e9-7899-47d1-a4f2-c161006db78b)
i just updated it to 10.0.9 and my rotation helper is on v10.0.33. The only addons i have are the MaxDps addons and letrix plus

**Comment by doadin** [comment]
@ArcStriker can you give me your talent loadout?

**Comment by ArcStriker** [comment]
<img width="774" alt="talents" src="https://github.com/kaminaris/MaxDps-Evoker/assets/148912192/6699b11a-3e32-42b0-933c-3579e20d4ee8">

**Comment by doadin** [comment]
@ArcStriker hard to copy like that was looking for the import/export sting, however I have figured out that the talent "Font of Magic" is modifying the spell id of both firebreath and Eternity Surge. I have just pushed an update to reflect this change with or without this talent. Hopefully there are no others that modify it, will find out with more testing. With so many talents its hard to test all possible builds and/or know what spells get modified by what(as far as it changing the ID) As I learn and note down all these it should be easier going forward, till blizz changes all the talents again lol.
